code/game/objects/items/devices/radio/electropack.dm

```diff
@@ -222,3 +223,81 @@ Code:
 			onclose(user, "radio")
 			return
 
+/obj/item/device/electropack/shockcollar/proc/remove_id()
+	if (id)
+		if (ismob(loc))
+			var/mob/M = loc
+			M.put_in_hands(id)
```

Comment:
what if this fails it should drop to ground or something

Reply:
Fixed I think.